paper_framework_sources/p1897_initial_executor_algorithms_for_cpp23.md

```diff
@@ -343,44 +364,43 @@ int r = sync_wait(just(3));
 The expression `execution::just(t...)` returns a sender, `s` wrapping the values `t...`.
 
  * If `t...` are nothrow movable then `execution::is_noexcept_sender(s)` shall be constexpr and return true.
- * When `execution::submit(s, r)` is called for some `r`, and r-value `s` will call `execution::set_value(r, std::move(t)...)`, inline with the caller.
- * When `execution::submit(s, r)` is called for some `r`, and l-value `s` will call `execution::set_value(r, t...)`, inline with the caller.
+ * When `execution::connect(s, r)` is called resulting in `operation_state` `o` and followed by `execution::start(o)` for some `r`, and r-value `s` will call `execution::set_value(r, std::move(t)...)`, inline with the caller.
```

Comment:
To be consistent with moving t the sender returned from just should be an rvalue, because the stored value will be moved out.

Would it be better to not put the move in to the description and we assume that that is QOI?